server/src/domain/person/person.repository.ts

```diff
@@ -10,7 +10,11 @@ export interface IPersonRepository {
   getAll(userId: string, options: PersonSearchOptions): Promise<PersonEntity[]>;
   getAllWithoutFaces(): Promise<PersonEntity[]>;
   getById(userId: string, personId: string): Promise<PersonEntity | null>;
-  getAssets(userId: string, id: string): Promise<AssetEntity[]>;
+
+  getAssets(userId: string, personId: string): Promise<AssetEntity[]>;
+  getAssetsCount(personId: string): Promise<number>;
+  deleteIdenticalAssets(primaryPersonId: string, mergePersonId: string): Promise<string[]>;
+  updateAssetsId(primaryPersonId: string, mergePersonId: string): Promise<number>;
```

Comment:
One of these might be a better name: (we aren't updating assets, we're updating faces)
- `reassignFaces(oldPersonId: string, newPersonId: string)`
- `transferFaces(oldPersonId: string, newPersonId: string)`
- `updateFaces(oldPersonId: string, newPersonId: string)`